testing/gnome-online-accounts/APKBUILD

```diff
@@ -1,40 +1,37 @@
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gnome-online-accounts
-pkgver=3.26.0
-pkgrel=1
-pkgdesc="client for GNOME web services"
-url="https://www.gnome.org/"
+pkgver=3.32.0
+pkgrel=0
+pkgdesc="Single sign-on framework for GNOME"
+url="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
 # limited by webkit2gtk
 arch="x86 x86_64 ppc64le aarch64"
-license="GPL"
-makedepends="gnome-desktop-dev libxslt itstool libxml2-utils 
-	webkit2gtk-dev json-glib-dev rest-dev libsecret-dev 
+license="GPL-2.0-or-later"
+makedepends="gnome-desktop-dev libxslt itstool libxml2-utils
+	webkit2gtk-dev json-glib-dev rest-dev libsecret-dev
 	docbook-xsl docbook-xml"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://download.gnome.org/sources/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
-builddir="$srcdir/$pkgname-$pkgver"
+subpackages="$pkgname-dev $pkgname-lang"
```

Comment:
What happened to the doc sub-package?

Reply:
Seems like g-o-a doesn't build any docs without gtk-doc anymore.